### DISCUSSION TITLE
does AGH verify DOH certificates?

### DISCUSSION THREAD

**Discussion by YBS-PC** [AdguardTeam/AdGuardHome#5641] [Q&A] [answered]
does AGH verify DOH certificates as mikrotik does?
![20200603223431-og35i](https://user-images.githubusercontent.com/52429295/227870213-14ee467b-8713-4f42-9f6e-e26a44d1404e.png)

**Comment by ainar-g** [comment, marked as answer]
Yes.  You can put something like `https://self-signed.badssl.com/`, test the upstreams, and see the `x509: certificate signed by unknown authority` error in the [verbose log](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog).
